examples/extensions-zalando/build.gradle

```diff
@@ -16,15 +16,14 @@ apply plugin: 'java'
 intellij {
     plugins = ['org.zalando.intellij.swagger:1.1.2', 'yaml']
     pluginName = 'intellij-swagger-extensions-zalando'
-    updateSinceUntilBuild = false
-    version = '2016.2'
```

Comment:
Wow… This was really a quite old version.

Reply:
Fortunately only in the example project :)